source/includes/destinationDataHandling-introduction.rst

```diff
@@ -0,0 +1,4 @@
+Starting in ``mongosync-beta`` 1.8, use the ``destinationDataHandling``
+option to define what happens if the destination cluster already has
```

Comment:
It might make sense to say "field in the start request" instead of "option" since this is no longer a command-line option? Depends on what the rest of the docs say.